(dp1
S'output'
p2
S"<type 'exceptions.SyntaxError'> invalid select attribute: shopkeeper_id"
p3
sS'layer'
p4
S'/home/stutisrivastava/web2py/applications/Test1/controllers/app.py'
p5
sS'code'
p6
S'# -*- coding: utf-8 -*-\n# try something like\n@service.json\ndef shopkeeper_login(shopkeeper_id):\n    shopkeeper = db(db.tbl_shopkeeper).select(shopkeeper_id = shopkeeper_id)\n    return dict(message="hello from app.py to "+str(shopkeeper))\n\ndef call():\n    session.forget()\n    return service()\n\nresponse._vars=response._caller(call)\n'
p7
sS'snapshot'
p8
(dp9
sS'traceback'
p10
S'Traceback (most recent call last):\n  File "/home/stutisrivastava/web2py/gluon/restricted.py", line 227, in restricted\n    exec ccode in environment\n  File "/home/stutisrivastava/web2py/applications/Test1/controllers/app.py", line 12, in <module>\n  File "/home/stutisrivastava/web2py/gluon/globals.py", line 412, in <lambda>\n    self._caller = lambda f: f()\n  File "/home/stutisrivastava/web2py/applications/Test1/controllers/app.py", line 10, in call\n    return service()\n  File "/home/stutisrivastava/web2py/gluon/tools.py", line 5954, in __call__\n    return self.serve_json(request.args[1:])\n  File "/home/stutisrivastava/web2py/gluon/tools.py", line 5636, in serve_json\n    s = universal_caller(self.json_procedures[args[0]], *args[1:], **d)\n  File "/home/stutisrivastava/web2py/gluon/tools.py", line 5295, in universal_caller\n    return f(**arg_dict)\n  File "/home/stutisrivastava/web2py/applications/Test1/controllers/app.py", line 5, in shopkeeper_login\n    shopkeeper = db(db.tbl_shopkeeper).select(shopkeeper_id = shopkeeper_id)\n  File "/home/stutisrivastava/web2py/gluon/packages/dal/pydal/objects.py", line 2025, in select\n    return adapter.select(self.query,fields,attributes)\n  File "/home/stutisrivastava/web2py/gluon/packages/dal/pydal/adapters/sqlite.py", line 124, in select\n    return super(SQLiteAdapter, self).select(query, fields, attributes)\n  File "/home/stutisrivastava/web2py/gluon/packages/dal/pydal/adapters/base.py", line 1280, in select\n    sql = self._select(query, fields, attributes)\n  File "/home/stutisrivastava/web2py/gluon/packages/dal/pydal/adapters/base.py", line 1091, in _select\n    raise SyntaxError(\'invalid select attribute: %s\' % key)\nSyntaxError: invalid select attribute: shopkeeper_id\n'
p11
s.